apps/judicial-system/backend/src/app/formatters/formatters.ts

```diff
@@ -165,11 +166,11 @@ export const formatDefenderResubmittedToCourtEmailNotification = (
   const subject = formatMessage(cf.subject, { courtCaseNumber })
   const body = formatMessage(cf.body, {
     courtCaseNumber,
-    courtName: court?.replace('dómur', 'dómi'),
+    courtName: court ? applyDativeCaseToCourtName(court) : 'héraðsdómi',
```

Comment:
We could also use the function for the default name (héraðsdómur) and also only define the adaptive case to court once after the subject var:
`const courtName = applyDativeCaseToCourtName(court || 'héraðsdómur')`
and use only courtName in line 169 and 173 below